src/index.md

```diff
@@ -2,7 +2,7 @@
 
 {{#include ./include/quiz-is-wip.md}}
 
-This is the unfair Rust quiz, a collection of very hard, gay, sometimes even unfair questions about Rust.
+This is the unfair Rust quiz, a collection of very hard, gay (in a good way, of course), and sometimes even unfair questions about Rust.
```

Comment:
while we are at it...

```suggestion
This is the unfair Rust quiz, a collection of extremely hard, gay (in a good way, of course), and sometimes even unfair questions about Rust.
```

Reply:
now I can't easily apply the suggestion because the line changed so I think I'll ignore it